e2e-tests/functions

```diff
@@ -231,96 +231,98 @@ apply_cluster() {
-				| yq w - 'spec.podAntiAffinity.pgBackRest' ${podAntiAffinity} \
-				| yq w - 'spec.podAntiAffinity.pgBouncer' ${podAntiAffinity} \
+			yq w ${tmp_dir}/percona-cr.yaml 'spec.pgPrimary.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.backup.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.pgBouncer.antiAffinityType' ${podAntiAffinity} \
 					>${tmp_dir}/cr.podAffinity.yaml
-			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/cr.yaml
+			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/percona-cr.yaml
 			;;
 		'required')
-			yq w ${tmp_dir}/cr.yaml 'spec.podAntiAffinity.default' ${podAntiAffinity} \
-				| yq w - 'spec.podAntiAffinity.pgBackRest' ${podAntiAffinity} \
-				| yq w - 'spec.podAntiAffinity.pgBouncer' ${podAntiAffinity} \
+			yq w ${tmp_dir}/percona-cr.yaml 'spec.pgPrimary.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.backup.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.pgBouncer.antiAffinityType' ${podAntiAffinity} \
 					>${tmp_dir}/cr.podAffinity.yaml
-			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/cr.yaml
+			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/percona-cr.yaml
 			;;
 		'disabled')
-			yq w ${tmp_dir}/cr.yaml 'spec.podAntiAffinity.default' ${podAntiAffinity} \
-				| yq w - 'spec.podAntiAffinity.pgBackRest' ${podAntiAffinity} \
-				| yq w - 'spec.podAntiAffinity.pgBouncer' ${podAntiAffinity} \
+			yq w ${tmp_dir}/percona-cr.yaml 'spec.pgPrimary.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.backup.antiAffinityType' ${podAntiAffinity} \
+				| yq w - 'spec.pgBouncer.antiAffinityType' ${podAntiAffinity} \
 					>${tmp_dir}/cr.podAffinity.yaml
-			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/cr.yaml
+			mv ${tmp_dir}/cr.podAffinity.yaml ${tmp_dir}/percona-cr.yaml
 			;;
 		*) ;;
 	esac
 
 	case ${backup} in
 		'local')
-			yq w -i "${tmp_dir}/cr.yaml" 'spec.backrestStorageTypes[0]' 'local'
+			yq w -i "${tmp_dir}/percona-cr.yaml" 'spec.backrestStorageTypes[0]' 'local'
 			;;
 		'gcs')
-			yq w "${tmp_dir}/cr.yaml" 'spec.backrestStorageTypes[0]' 'gcs' \
-				| yq w - 'spec.backrestGCSBucket' ${BUCKET} \
+			yq w "${tmp_dir}/percona-cr.yaml" 'spec.backup.storageTypes[0]' 'gcs' \
+				| yq w - 'spec.backup.storages[s3].bucket' ${BUCKET} \
 					>"${tmp_dir}/cr.backup.yaml"
-			mv "${tmp_dir}/cr.backup.yaml" "${tmp_dir}/cr.yaml"
+			mv "${tmp_dir}/cr.backup.yaml" "${tmp_dir}/percona-cr.yaml"
 			;;
 		's3')
-			yq w "${tmp_dir}/cr.yaml" 'spec.backrestS3Bucket' ${BUCKET} \
-				| yq w - 'spec.backrestS3Endpoint' 's3.amazonaws.com' \
-				| yq w - 'spec.backrestS3Region' 'us-east-1' \
-				| yq w - 'spec.backrestStorageTypes[0]' 's3' \
+			yq w "${tmp_dir}/percona-cr.yaml" 'spec.backup.storages[s3].bucket' ${BUCKET} \
+				| yq w - 'spec.backup.storages[s3].endpointUrl' 's3.amazonaws.com' \
+				| yq w - 'spec.backup.storages[s3].region' 'us-east-1' \
+				| yq w - 'spec.backup.storages[s3].uriStyle' 'path' \
+				| yq w - 'spec.backup.storages[s3].verifyTLS' 'false' \
```

Comment:
why do we need this here?

It was needed for minio gw only since it had self-signed certificate

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/6af9b2389e856fb97600ea59b222f3ae1b4d9a38